lib/build/bundle.js

```diff
@@ -438,26 +432,24 @@ exports.Bundle = class {
     return 'function _aureliaConfigureModuleLoader(){' + config + '}';
   }
 
-  setIndexFileConfigTarget(platform, location) {
-    //Replace the reference to the vendor bundle in the "index.html" file to use the correct build revision file (or remove the build revision hash);
-    const createSrcFileRegex = require('./utils').createSrcFileRegex;
-    let indexLocation = platform.index;
+  async writeRevToIndex(platform) {
     try {
-      let indexFile = fs.readFileSync(indexLocation);
-      let outputDir = platform.baseUrl || platform.output; //If we have a baseUrl, then the files are served from there, else it's the output
-      let configMatcher = createSrcFileRegex(outputDir, this.moduleId);
-      if (configMatcher.test(indexFile)) {
-        indexFile = indexFile.replace(configMatcher, 'src="$2' + location + '"');//Replace the old reference to the file with whatever the new one is (preserving any unknown path parts before)
-        fs.writeFile(indexLocation, indexFile);
-      } else {
-        logger.error('Error: Unable to update ' + this.moduleId + ' path to ' + location + ', could not find existing reference to replace');
-      }
-    } catch (err) {
-      if (err.code === 'ENOENT') {
-        logger.error('Error: No index file found at "' + indexLocation + '"');
-      } else {
-        logger.error(err);
+      // Validate rev is enabled and a hash was generated
+      if (platform.index && this.hash) {
```

Comment:
Found a bug, you need to write empty hash to index file.
Consider:
```sh
au run --env prod
# terminate it, then
au run
```

The second `au run` need to overwrite index file to empty hash.